website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -0,0 +1,246 @@
+challenges in terms of maintenance - most notably, they need to run constantly,
+be configured (including authentication and connectivity to the storage
+platform), and updated to keep track with new features and bug fixes and keep
+compatibility with the underlying storage platform. They also introduce a couple
+of moving parts, can be difficult to troubleshoot, and have a complex security
+profile (due to needing to run as `privileged` containers in order to be able to
+mount volumes).
+
+The [Stateful Workloads with CSI
+tutorial](/nomad/tutorials/stateful-workloads/stateful-workloads-csi-volumes)
+and the [Nomad CSI demo
+repository](https://github.com/hashicorp/nomad/tree/main/demo/csi) offer
+guidance and examples on how to use CSI plugins with Nomad and include job files
+for running the plugins and configuration files for creating and consuming
+volumes.
+
+### Host volumes
+
+Host volumes mount paths from the host (the Nomad client) into allocations.
+Nomad is aware of host volume availability and makes use of it for job
+scheduling. However, Nomad does not know about the volume's underlying
+characteristics, such as if it is a standard folder on a local ext4 filesystem,
+backed by a distributed networked storage such as GlusterFS, or a mounted
+NFS/CIFS volume from a NAS or a public cloud service such as AWS EFS. Therefore
+you can use host volumes for both local somewhat persistent storage and for
+highly persistent networked storage.
+
+Because you need to declare host volumes in the Nomad agent's configuration
+file, you must restart the Nomad client to reconfigure them. This makes host
+volumes impractical if you frequently change your storage configuration.
+Furthermore, it might require coordination between different
+[personnas](/nomad/docs/concepts/security#personas) to configure and consume
+host volumes. For example, a Nomad Administrator must modify Nomad's
+configuration file to add/update/remove host volumes to make them available for
+consumption by Nomad Operators. Or, with networked host volumes, a Storage
+Administrator will need to provision the volumes and make them available to the
+Nomad clients. A System Administrator will then mount them on the Nomad clients.
+
+Host volumes backed by local storage help persist data that is not critical, for
+example an on-disk cache that can be rebuilt if needed. When backed by networked
+storage such as NFS/CIFS-mounted volumes or distributed storage via
+GlusterFS/Ceph, host volumes provide a quick option to consume highly available
+and reliable storage.
+
+Refer to the [Stateful workloads with Nomad host
+volumes](/nomad/tutorials/stateful-workloads/stateful-workloads-host-volumes)
+tutorial to learn more about using host volumes with Nomad.
+
+#### NFS caveats
+
+A few caveats with NFS-backed host volumes include ACLs, reliability, and
+performance. NFS mount options should be the same on all mounting Nomad clients.
```

Comment:
This doesn't seem to be well-described here, but probably goes better with the paragraph about UIDs. I'd consider moving this down or moving the block size discussion after the UID discussion.